src/packages/galaxy/galaxy.py

```diff
@@ -67,12 +67,17 @@ def __init__(self, url: str = None):
             A ``StarSystem`` object representing the found system, or ``None`` if none was found.
         """
 
-        data = await self._call("search", {"name": name, "limit": 1})
-        if 'data' in data and data['data'] and data['data'][0]['name'].casefold() == name.casefold():
+        data = await self._call("api/systems", {
+            "filter[name:ilike]": name,
+            "sort": "name",
+            "limit": 1
+        })
+        print(data)
```

Comment:
[error] logging.debug ?
```suggestion
        logger.debug("system API response := {}", data)
```